packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -57,6 +68,7 @@ const getReactRouteGroups = (showReactApp, reactRoute) => {
         'signin_verified',
         'signin_bounced',
       ]),
+      fullProdRollout: false,
```

Comment:
Can we add this to the `ReactRouteGroup` type in `react-app/types.ts`? It only helps for development so doesn't address Peter's comment but can still be helpful.